data/sf.yaml

```diff
@@ -3,7 +3,7 @@ info:
   city: San Francisco
   link: san-francisco
   date: 11/02/2019
-  hour: '15:00'
+  hour: '18:00'
```

Comment:
we probably want to show both, right? a la `'15:00 (stream) 18:00 (live talks)` or something?

Reply:
Good point! Sara says this might be just text, so let's try this 😅 , updated